src/interfaces/highs_c_api.h

```diff
@@ -25,110 +90,115 @@ extern "C" {
 /**
  * Formulate and solve a linear program using HiGHS.
  *
- * @param numcol    the number of columns
- * @param numrow    the number of rows
- * @param numnz     the number of nonzeros in the constraint matrix
- * @param a_format  the format of the constraint matrix
- * @param sense     the optimization sense. Use -1 for maximization, 1 otherwise
+ * @param num_col   the number of columns
+ * @param num_row   the number of rows
+ * @param num_nz    the number of nonzeros in the constraint matrix
+ * @param a_format  the format of the constraint matrix as a `HighsMatrixFormat`
```

Comment:
```suggestion
 * @param a_format  the format of the constraint matrix as a `kHighsMatrixFormat`
```

We should use however we chose to name this. There are lots of other instances of enums as well.